infra/terraform/modules/aws_account_logging/lambda.tf

```diff
@@ -0,0 +1,62 @@
+# Install dependencies
+resource "null_resource" "s3logs_install_deps" {
+    provisioner "local-exec" {
+        command = "${path.module}/s3logs/build.sh"
+    }
+
+    triggers {
```

Comment:
As discussed earlier, `triggers` don't seem to work as we thought so let's just replace force the rebuild all the times until we find a better solution: https://github.com/ministryofjustice/analytics-platform-ops/pull/89/commits/19046f0c10da4cde551bdc6f0847e332b0d92a7d